paper.tex

```diff
@@ -260,17 +251,18 @@ \subsection*{Population genetics model}
 (i.e., ``soft'' selection~\citep{christiansen1975hard}).
 While these restrictions preclude some population-genetic scenarios of
 interest, such as continuous spatial structure
-\aprcomment{citations here, and revisit} or
-\aprcomment{stochastic carrying-capacity models, or something like that},
+\citep{wright1943isolation,barton2002neutral,ringbauer2017inferring,battey2020space}
```

Comment:
Can you cite this paper as well please? https://pubmed.ncbi.nlm.nih.gov/20408876/

It would look weird if it was left out and I'm an author here.

Reply:
Yes, of course! I'll add it.